src/UnitTests/Core/Impl/Mef/IExportProvider.cs

```diff
@@ -3,10 +3,9 @@
 
 using System;
 using System.Collections.Generic;
-using Microsoft.UnitTests.Core.XUnit;
 
 namespace Microsoft.UnitTests.Core.Mef {
-    public interface IExportProvider : IMethodFixture {
+    public interface IExportProvider {
         T GetExportedValue<T>() where T : class;
```

Comment:
You can leave only `T GetExportedValue<T>() where T : class;` - all other methods can be removed